diff --git a/config/templates/config/templates/gnu/compiler-compat-g++.h b/config/templates/gnu/compiler-compat-g++.h
index 0284f38..bcbeb60 100644
--- a/config/templates/gnu/compiler-compat-g++.h
+++ b/config/templates/gnu/compiler-compat-g++.h
@@ -149,3 +149,7 @@ bool __builtin_mul_overflow(type1 x, type2 y, rtype *ptr);



+// Coverity Analysis does not seem recognize _Float128 now used by GCC in f27+
+#if __GNUC__ == 7 && __GNUC_MINOR__ >= 1
+#define _Float128 long double
+#endif
diff --git a/config/templates/gnu/compiler-compat-gcc.h b/config/templates/gnu/compiler-compat-gcc.h
index 35bfebd..f955ba6 100644
--- a/config/templates/gnu/compiler-compat-gcc.h
+++ b/config/templates/gnu/compiler-compat-gcc.h
@@ -64,3 +64,7 @@ extern int __builtin_mul_overflow(int, int, int *);



+// Coverity Analysis does not seem recognize _Float128 now used by GCC in f27+
+#if __GNUC__ == 7 && __GNUC_MINOR__ >= 1
+#define _Float128 long double
+#endif
