--- 20-acpi-vendor.hwdb.base	2026-02-17 11:46:19.559676127 +0100
+++ 20-acpi-vendor.hwdb	2026-02-17 11:46:19.570676054 +0100
@@ -3,6 +3,8 @@
 # Data imported from:
 #     https://uefi.org/uefi-pnp-export
 #     https://uefi.org/uefi-acpi-export
+#
+# With various additions from other sources
 
 acpi:3GVR*:
  ID_VENDOR_FROM_DATABASE=VR Technology Holdings Limited
@@ -448,6 +450,9 @@
 acpi:AAA*:
  ID_VENDOR_FROM_DATABASE=Avolites Ltd
 
+acpi:AAC*:
+ ID_VENDOR_FROM_DATABASE=AcerView
+
 acpi:AAE*:
  ID_VENDOR_FROM_DATABASE=Anatek Electronics Inc.
 
@@ -475,6 +480,9 @@
 acpi:ABO*:
  ID_VENDOR_FROM_DATABASE=D-Link Systems Inc
 
+acpi:ABP*:
+ ID_VENDOR_FROM_DATABASE=Advansys
+
 acpi:ABS*:
  ID_VENDOR_FROM_DATABASE=Abaco Systems, Inc.
 
@@ -520,7 +528,7 @@
 acpi:ACO*:
  ID_VENDOR_FROM_DATABASE=Allion Computer Inc.
 
-acpi:ACP*:
+acpi:ACP[0-9A-F]*:
  ID_VENDOR_FROM_DATABASE=Aspen Tech Inc
 
 acpi:ACR*:
@@ -799,6 +807,9 @@
 acpi:AMT*:
  ID_VENDOR_FROM_DATABASE=AMT International Industry
 
+acpi:AMW*:
+ ID_VENDOR_FROM_DATABASE=AMW
+
 acpi:AMX*:
  ID_VENDOR_FROM_DATABASE=AMX LLC
 
@@ -847,6 +858,9 @@
 acpi:AOA*:
  ID_VENDOR_FROM_DATABASE=AOpen Inc.
 
+acpi:AOC*:
+ ID_VENDOR_FROM_DATABASE=AOC
+
 acpi:AOE*:
  ID_VENDOR_FROM_DATABASE=Advanced Optics Electronics, Inc.
 
@@ -856,6 +870,9 @@
 acpi:AOT*:
  ID_VENDOR_FROM_DATABASE=Alcatel
 
+acpi:APA*:
+ ID_VENDOR_FROM_DATABASE=Adaptec
+
 acpi:APC*:
  ID_VENDOR_FROM_DATABASE=American Power Conversion
 
@@ -1037,7 +1054,7 @@
  ID_VENDOR_FROM_DATABASE=ALPS ALPINE CO., LTD.
 
 acpi:AUO*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - AUO
+ ID_VENDOR_FROM_DATABASE=AU Optronics
 
 acpi:AUR*:
  ID_VENDOR_FROM_DATABASE=Aureal Semiconductor
@@ -1117,6 +1134,9 @@
 acpi:AXE*:
  ID_VENDOR_FROM_DATABASE=Axell Corporation
 
+acpi:AXE*:
+ ID_VENDOR_FROM_DATABASE=D-Link Systems Inc
+
 acpi:AXI*:
  ID_VENDOR_FROM_DATABASE=American Magnetics
 
@@ -1276,6 +1296,9 @@
 acpi:BML*:
  ID_VENDOR_FROM_DATABASE=BIOMED Lab
 
+acpi:BMM*:
+ ID_VENDOR_FROM_DATABASE=BMM
+
 acpi:BMS*:
  ID_VENDOR_FROM_DATABASE=BIOMEDISYS
 
@@ -1288,6 +1311,9 @@
 acpi:BNO*:
  ID_VENDOR_FROM_DATABASE=Bang & Olufsen
 
+acpi:BNQ*:
+ ID_VENDOR_FROM_DATABASE=BenQ Corporation
+
 acpi:BNS*:
  ID_VENDOR_FROM_DATABASE=Boulder Nonlinear Systems
 
@@ -1534,6 +1560,9 @@
 acpi:CHA*:
  ID_VENDOR_FROM_DATABASE=Chase Research PLC
 
+acpi:CHC*:
+ ID_VENDOR_FROM_DATABASE=Chic Technology Corp.
+
 acpi:CHD*:
  ID_VENDOR_FROM_DATABASE=ChangHong Electric Co.,Ltd
 
@@ -1699,6 +1728,9 @@
 acpi:COD*:
  ID_VENDOR_FROM_DATABASE=CODAN Pty. Ltd.
 
+acpi:COG*:
+ ID_VENDOR_FROM_DATABASE=Cogent
+
 acpi:COI*:
  ID_VENDOR_FROM_DATABASE=Codec Inc.
 
@@ -2117,7 +2149,7 @@
  ID_VENDOR_FROM_DATABASE=Dragon Information Technology
 
 acpi:DJE*:
- ID_VENDOR_FROM_DATABASE=Capstone Visua lProduct Development
+ ID_VENDOR_FROM_DATABASE=Capstone Visual Product Development
 
 acpi:DJP*:
  ID_VENDOR_FROM_DATABASE=Maygay Machines, Ltd
@@ -2470,6 +2502,9 @@
 acpi:EIN*:
  ID_VENDOR_FROM_DATABASE=Elegant Invention
 
+acpi:EIZ*:
+ ID_VENDOR_FROM_DATABASE=Eizo
+
 acpi:EKA*:
  ID_VENDOR_FROM_DATABASE=MagTek Inc.
 
@@ -2740,6 +2775,9 @@
 acpi:FCG*:
  ID_VENDOR_FROM_DATABASE=First International Computer Ltd
 
+acpi:FCM*:
+ ID_VENDOR_FROM_DATABASE=Funai
+
 acpi:FCS*:
  ID_VENDOR_FROM_DATABASE=Focus Enhancements, Inc.
 
@@ -3116,7 +3154,7 @@
  ID_VENDOR_FROM_DATABASE=General Standards Corporation
 
 acpi:GSM*:
- ID_VENDOR_FROM_DATABASE=Goldstar Company Ltd
+ ID_VENDOR_FROM_DATABASE=LG Electronics
 
 acpi:GSN*:
  ID_VENDOR_FROM_DATABASE=Grandstream Networks, Inc.
@@ -3226,6 +3264,9 @@
 acpi:HEC*:
  ID_VENDOR_FROM_DATABASE=Hisense Electric Co., Ltd.
 
+acpi:HEI*:
+ ID_VENDOR_FROM_DATABASE=Hyundai
+
 acpi:HEL*:
  ID_VENDOR_FROM_DATABASE=Hitachi Micro Systems Europe Ltd
 
@@ -3361,6 +3402,9 @@
 acpi:HSD*:
  ID_VENDOR_FROM_DATABASE=HannStar Display Corp
 
+acpi:HSL*:
+ ID_VENDOR_FROM_DATABASE=Hansol
+
 acpi:HSM*:
  ID_VENDOR_FROM_DATABASE=AT&T Microelectronics
 
@@ -3487,6 +3531,9 @@
 acpi:ICI*:
  ID_VENDOR_FROM_DATABASE=Infotek Communication Inc
 
+acpi:ICL*:
+ ID_VENDOR_FROM_DATABASE=Fujitsu ICL
+
 acpi:ICM*:
  ID_VENDOR_FROM_DATABASE=Intracom SA
 
@@ -3583,6 +3630,9 @@
 acpi:IKE*:
  ID_VENDOR_FROM_DATABASE=Ikegami Tsushinki Co. Ltd.
 
+acpi:IKN*:
+ ID_VENDOR_FROM_DATABASE=IKON
+
 acpi:IKS*:
  ID_VENDOR_FROM_DATABASE=Ikos Systems Inc
 
@@ -3631,6 +3681,9 @@
 acpi:IMX*:
  ID_VENDOR_FROM_DATABASE=arpara Technology Co., Ltd.
 
+acpi:IMS*:
+ ID_VENDOR_FROM_DATABASE=Integrated Micro Solution Inc.
+
 acpi:INA*:
  ID_VENDOR_FROM_DATABASE=Inventec Corporation
 
@@ -4159,6 +4212,9 @@
 acpi:LAN*:
  ID_VENDOR_FROM_DATABASE=Sodeman Lancom Inc
 
+acpi:LAP*:
+ ID_VENDOR_FROM_DATABASE=BenQ
+
 acpi:LAS*:
  ID_VENDOR_FROM_DATABASE=LASAT Comm. A/S
 
@@ -4210,6 +4266,9 @@
 acpi:LED*:
  ID_VENDOR_FROM_DATABASE=Long Engineering Design Inc
 
+acpi:LED*:
+ ID_VENDOR_FROM_DATABASE=LeafNet
+
 acpi:LEG*:
  ID_VENDOR_FROM_DATABASE=Legerity, Inc
 
@@ -4228,6 +4287,9 @@
 acpi:LGD*:
  ID_VENDOR_FROM_DATABASE=LG Display
 
+acpi:LGD*:
+ ID_VENDOR_FROM_DATABASE=LG Display
+
 acpi:LGI*:
  ID_VENDOR_FROM_DATABASE=Logitech Inc
 
@@ -4294,6 +4356,9 @@
 acpi:LND*:
  ID_VENDOR_FROM_DATABASE=Land Computer Company Ltd
 
+acpi:LNE*:
+ ID_VENDOR_FROM_DATABASE=Linksys
+
 acpi:LNK*:
  ID_VENDOR_FROM_DATABASE=Link Tech Inc
 
@@ -4328,7 +4393,7 @@
  ID_VENDOR_FROM_DATABASE=Design Technology
 
 acpi:LPL*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - LPL
+ ID_VENDOR_FROM_DATABASE=LG Philips
 
 acpi:LSC*:
  ID_VENDOR_FROM_DATABASE=LifeSize Communications
@@ -4504,6 +4569,9 @@
 acpi:MCX*:
  ID_VENDOR_FROM_DATABASE=Millson Custom Solutions Inc.
 
+acpi:MCY*:
+ ID_VENDOR_FROM_DATABASE=Microdyne
+
 acpi:MDA*:
  ID_VENDOR_FROM_DATABASE=Media4 Inc
 
@@ -4750,6 +4818,9 @@
 acpi:MOM*:
  ID_VENDOR_FROM_DATABASE=Momentum Data Systems
 
+acpi:MON*:
+ ID_VENDOR_FROM_DATABASE=Daewoo
+
 acpi:MOS*:
  ID_VENDOR_FROM_DATABASE=Moses Corporation
 
@@ -4990,6 +5061,9 @@
 acpi:NAL*:
  ID_VENDOR_FROM_DATABASE=Network Alchemy
 
+acpi:NAN*:
+ ID_VENDOR_FROM_DATABASE=Nanao
+
 acpi:NAT*:
  ID_VENDOR_FROM_DATABASE=NaturalPoint Inc.
 
@@ -5530,6 +5604,9 @@
 acpi:PCX*:
  ID_VENDOR_FROM_DATABASE=PC Xperten
 
+acpi:PDC*:
+ ID_VENDOR_FROM_DATABASE=Polaroid
+
 acpi:PDM*:
  ID_VENDOR_FROM_DATABASE=Psion Dacom Plc.
 
@@ -5593,9 +5670,6 @@
 acpi:PHE*:
  ID_VENDOR_FROM_DATABASE=Philips Medical Systems Boeblingen GmbH
 
-acpi:PHI*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - PHI
-
 acpi:PHL*:
  ID_VENDOR_FROM_DATABASE=Philips Consumer Electronics Company
 
@@ -5686,9 +5760,6 @@
 acpi:PNL*:
  ID_VENDOR_FROM_DATABASE=Panelview, Inc.
 
-acpi:PNP*:
- ID_VENDOR_FROM_DATABASE=Microsoft
-
 acpi:PNR*:
  ID_VENDOR_FROM_DATABASE=Planar Systems, Inc.
 
@@ -6166,9 +6237,6 @@
 acpi:RTI*:
  ID_VENDOR_FROM_DATABASE=Rancho Tech Inc
 
-acpi:RTK*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - RTK
-
 acpi:RTL*:
  ID_VENDOR_FROM_DATABASE=Realtek Semiconductor Company Ltd
 
@@ -6343,9 +6411,6 @@
 acpi:SEE*:
  ID_VENDOR_FROM_DATABASE=SeeColor Corporation
 
-acpi:SEG*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - SEG
-
 acpi:SEI*:
  ID_VENDOR_FROM_DATABASE=Seitz & Associates Inc
 
@@ -6829,6 +6894,9 @@
 acpi:SVD*:
  ID_VENDOR_FROM_DATABASE=SVD Computer
 
+acpi:SVE*:
+ ID_VENDOR_FROM_DATABASE=SVEC
+
 acpi:SVI*:
  ID_VENDOR_FROM_DATABASE=Sun Microsystems
 
@@ -6913,6 +6981,9 @@
 acpi:SZM*:
  ID_VENDOR_FROM_DATABASE=Shenzhen MTC Co., Ltd
 
+acpi:SZV*:
+ ID_VENDOR_FROM_DATABASE=OvisLink
+
 acpi:TAA*:
  ID_VENDOR_FROM_DATABASE=Tandberg
 
@@ -7003,6 +7074,9 @@
 acpi:TDG*:
  ID_VENDOR_FROM_DATABASE=Six15 Technologies
 
+acpi:TDK*:
+ ID_VENDOR_FROM_DATABASE=TDK USA Corporation
+
 acpi:TDM*:
  ID_VENDOR_FROM_DATABASE=Tandem Computer Europe Inc
 
@@ -7045,6 +7119,9 @@
 acpi:TEV*:
  ID_VENDOR_FROM_DATABASE=Televés, S.A.
 
+acpi:TEX*:
+ ID_VENDOR_FROM_DATABASE=Texas Instruments
+
 acpi:TEZ*:
  ID_VENDOR_FROM_DATABASE=Tech Source Inc.
 
@@ -7174,9 +7251,6 @@
 acpi:TNC*:
  ID_VENDOR_FROM_DATABASE=TNC Industrial Company Ltd
 
-acpi:TNJ*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - TNJ
-
 acpi:TNM*:
  ID_VENDOR_FROM_DATABASE=TECNIMAGEN SA
 
@@ -7489,14 +7563,14 @@
 acpi:UNC*:
  ID_VENDOR_FROM_DATABASE=Unisys Corporation
 
-acpi:UND*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - UND
+acpi:UND*
+ ID_VENDOR_FROM_DATABASE=Unisys Corporation
 
-acpi:UNE*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - UNE
+acpi:UNE*
+ ID_VENDOR_FROM_DATABASE=Unisys Corporation
 
-acpi:UNF*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - UNF
+acpi:UNF*
+ ID_VENDOR_FROM_DATABASE=Unisys Corporation
 
 acpi:UNI*:
  ID_VENDOR_FROM_DATABASE=Uniform Industry Corp.
@@ -7531,6 +7605,9 @@
 acpi:USA*:
  ID_VENDOR_FROM_DATABASE=Utimaco Safeware AG
 
+acpi:USC*:
+ ID_VENDOR_FROM_DATABASE=UltraStor
+
 acpi:USD*:
  ID_VENDOR_FROM_DATABASE=U.S. Digital Corporation
 
@@ -7792,9 +7869,6 @@
 acpi:WAL*:
  ID_VENDOR_FROM_DATABASE=Wave Access
 
-acpi:WAN*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - WAN
-
 acpi:WAV*:
  ID_VENDOR_FROM_DATABASE=Wavephore
 
@@ -7922,7 +7996,7 @@
  ID_VENDOR_FROM_DATABASE=WyreStorm Technologies LLC
 
 acpi:WYS*:
- ID_VENDOR_FROM_DATABASE=Myse Technology
+ ID_VENDOR_FROM_DATABASE=Wyse Technology
 
 acpi:WYT*:
  ID_VENDOR_FROM_DATABASE=Wooyoung Image & Information Co.,Ltd.
@@ -7936,9 +8010,6 @@
 acpi:XDM*:
  ID_VENDOR_FROM_DATABASE=XDM Ltd.
 
-acpi:XER*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - XER
-
 acpi:XES*:
  ID_VENDOR_FROM_DATABASE=Extreme Engineering Solutions, Inc.
 
@@ -7972,9 +8043,6 @@
 acpi:XNT*:
  ID_VENDOR_FROM_DATABASE=XN Technologies, Inc.
 
-acpi:XOC*:
- ID_VENDOR_FROM_DATABASE=DO NOT USE - XOC
-
 acpi:XQU*:
  ID_VENDOR_FROM_DATABASE=SHANGHAI SVA-DAV ELECTRONICS CO., LTD
 
@@ -8041,6 +8109,9 @@
 acpi:ZBX*:
  ID_VENDOR_FROM_DATABASE=Zebax Technologies
 
+acpi:ZCM*:
+ ID_VENDOR_FROM_DATABASE=Zenith
+
 acpi:ZCT*:
  ID_VENDOR_FROM_DATABASE=ZeitControl cardsystems GmbH
 
